src/Reflection/FunctionReflection.php

```diff
@@ -223,6 +223,18 @@ public function getParameters(): array
 					true
 				);
 			}
+
+			if (
+				$this->reflection->getName() === 'get_defined_functions'
+				&& PHP_VERSION_ID >= 70110
```

Comment:
If it's fixed in 7.1.14, this constant is wrong - should be `70114`. But I'd rather have a different condition here - one that checks whether the first parameter is optional or not, rather than checking for a version (which is more brittle).

Reply:
So would be like: 
```php
if (
    $this->reflection->getName() === 'get_defined_functions'
    && $this->reflection->getNumberOfRequiredParameters() === 1
    && count($this->attributes) === 0
)
```
?